src/Student/StudentTest.java

```diff
@@ -0,0 +1,23 @@
+package Student;
+
+import Student.Student;
+
+public class StudentTest {
+    public static int arithmeticMeanScore(int mAverGrade1, int eAverGrade1, int forLangAverGrade1) {
+        int a = mAverGrade1;
```

Comment:
It's not needed to declare new variables a, b, c - you can to sum mAverGrade1, eAverGrade1, forLangAverGrade1 and divide on 3 at once